dsc_lib/src/configure/mod.rs

```diff
@@ -67,9 +67,21 @@ pub fn add_resource_export_results_to_configuration(resource: &DscResource, conf
     } else {
         for (i, instance) in export_result.actual_state.iter().enumerate() {
             let mut r = config_doc::Resource::new();
+            let mut props: Map<String, Value> = serde_json::from_value(instance.clone())?;
+            if let Some(kind) = props.remove("_kind") {
+                r.kind = kind.as_str().map(std::string::ToString::to_string);
```

Comment:
This will fail if a resource incorrectly returns `_kind` as something other than a string